sdk/communication/azure-communication-sms/README.md

```diff
@@ -21,102 +21,126 @@ Azure Communication SMS is used to send simple text messages.
 <dependency>
   <groupId>com.azure</groupId>
   <artifactId>azure-communication-sms</artifactId>
-  <version>1.0.0-beta.4</version> 
+  <version>1.0.0-beta.4</version>
 </dependency>
 ```
 
-## Key concepts
-
-There are two different forms of authentication to use the Azure Communication SMS Service.
+## Authenticate the client
 
 ### Azure Active Directory Token Authentication
+A `DefaultAzureCredential` object must be passed to the `SmsClientBuilder` via the credential() function. Endpoint and httpClient must also be set via the endpoint() and httpClient() functions respectively.
```

Comment:
I'm ok using the DefaultAzureCredential as our example, but this statement makes me think we can only use DefaultAzureCredential. The .net readme has "SMS clients can also be authenticated using a valid token credential." So maybe something more like that?